src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,51 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+}) => (
+  <>
```

Comment:
You have custom component as base. You don't need to use fragment here.

Use react fragments only when you need to control parent tag of the component.